crates/ubrn_cli/src/config/mod.rs

```diff
@@ -59,6 +59,16 @@ fn trim_react_native(name: &str) -> String {
     name.trim_matches('-').trim_matches('_').to_string()
 }
 
+fn trim_react_native_2(name: &str) -> String {
+    name.strip_prefix("RN")
+        .unwrap_or(name)
+        .replace("ReactNative", "")
+        .replace("react-native", "")
+        .trim_matches('-')
+        .trim_matches('_')
+        .to_string()
+}
```

Comment:
I'm going to start __slowly__ work out where this is needed and where it isn't. Selectively adding `trim_react_native_2` to defaults until everything works.